sfopenboson/tests/test_bose_hubbard_prop.py

```diff
@@ -230,77 +252,35 @@ def test_circulant(self):
 
         tol = 1e-1
 
-        self.assertTrue(np.allclose(state.fock_prob([2, 0, 0]), exp[0], rtol=tol))
-        self.assertTrue(np.allclose(state.fock_prob([1, 1, 0]), exp[1], rtol=tol))
-        self.assertTrue(np.allclose(state.fock_prob([1, 0, 1]), exp[2], rtol=tol))
-        self.assertTrue(np.allclose(state.fock_prob([0, 2, 0]), exp[3], rtol=tol))
-        self.assertTrue(np.allclose(state.fock_prob([0, 1, 1]), exp[4], rtol=tol))
-        self.assertTrue(np.allclose(state.fock_prob([0, 0, 2]), exp[5], rtol=tol))
+        assert np.allclose(state.fock_prob([2, 0, 0]), exp[0], rtol=tol)
+        assert np.allclose(state.fock_prob([1, 1, 0]), exp[1], rtol=tol)
+        assert np.allclose(state.fock_prob([1, 0, 1]), exp[2], rtol=tol)
+        assert np.allclose(state.fock_prob([0, 2, 0]), exp[3], rtol=tol)
+        assert np.allclose(state.fock_prob([0, 1, 1]), exp[4], rtol=tol)
+        assert np.allclose(state.fock_prob([0, 0, 2]), exp[5], rtol=tol)
 
 
-class TestToolchain(BaseTest):
+class TestToolchain:
     """Tests for the BoseHubbardPropagation toolchain"""
-    def setUp(self):
-        """parameters"""
-        self.hbar = 2.
-        self.eng, _ = sf.Engine(2, hbar=self.hbar)
-        self.J = -1
-        self.U = 1.5
-        self.t = 1.086
-        self.k = 20
-        self.tol = 1e-2
-
-        self.H = bose_hubbard(1, 2, self.J, self.U)
-        self.Hquad = get_quad_operator(self.H, hbar=self.hbar)
-
-    def test_hbar_outside_context(self):
```

Comment:
no longer need these hbar tests?

Reply:
No, this is because of the change in how `hbar` is handled in SF v0.11; hbar is no longer part of the engine/program context, but a separate global variable.